common/determined_common/experimental/checkpoint/_torch.py

```diff
@@ -20,13 +20,16 @@ def load_model(
 
     trial_context = cast(PyTorchTrialContext, trial_context)
     trial = cast(PyTorchTrial, trial_cls(trial_context))
-
     if "model_state_dict" in checkpoint:
         # Backward compatible with older checkpoint format.
         model = trial.build_model()
         model.load_state_dict(checkpoint["model_state_dict"])
         return model
     else:
-        for idx, model in enumerate(trial_context.models):
-            model.load_state_dict(checkpoint["models_state_dict"][idx])
-        return trial
+        model = trial.build_model()
+        model.load_state_dict(checkpoint["models_state_dict"][0])
+        return model
+        # TODO: Fix for multiple model case
```

Comment:
question: Do we have a ticket?

Reply:
Haven't yet made the ticket, but we should do that.